README.md

```diff
@@ -1,4 +1,262 @@
+feature as a pull request to the corresponding repo. Being "Accepted" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "Accepted"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether an IPFS developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+### Modifications to Accepted RFCs
+
+Modifications to "Accepted" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next major release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the working group to decide.
+
+_If_ you make changes to an RFC after it's been accepted, add a note in the header of the RFC with a very short summary of the change and links to any relevant discussions. This way the change is very explicit and hard to miss.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the working group may schedule meetings with the
+author and/or relevant stakeholders to discuss the issues in greater detail,
+and in some cases the topic may be discussed at a working group meeting. In either
+case a summary from the meeting will be posted back to the RFC pull request.
+
+A working group makes final decisions about RFCs after the benefits and drawbacks
+are well understood. These decisions can be made at any time, but the working group
+will regularly issue decisions. When a decision is made, the RFC pull request
+will either be merged or closed. In either case, if the reasoning is not clear
+from the discussion in thread, the working group will add a comment describing the
+rationale for the decision.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right
+away. Other accepted RFCs can represent features that can wait until some
+arbitrary developer feels like doing the work. Every accepted RFC has an
+associated issue tracking its implementation in the IPFS repository; thus that
```

Comment:
What's the "the IPFS repository"?

Reply:
✅  changed to "the corresponding IPFS repository"